### DISCUSSION TITLE
Handling certain less popular scripts

### DISCUSSION THREAD

**Discussion by chakrabortydeepro** [PanditProject/Website#25] [Help] [answered]
@AmirSimantov I found that the project website cannot handle certain less popular scripts such as Grantha, Sharada. Newa or Tirhuta. Sometimes, we need to put titles in these scripts in the section "name in local script". But when these scripts in Unicode are used it gives the error message "The website encountered an unexpected error. Please try again later.". 

Is there any way to fix this problem?

**Comment by chakrabortydeepro** [comment]
@AmirSimantov any thought about it, Amir? Indian scripts do help to localize entities. Allowing all sorts of Unicode types will be really very useful.

**Comment by AmirSimantov** [comment]
@chakrabortydeepro, sorry for the late reply, I have to talk to @ybronner about it, and our resources are limited right now.
Could you please list here all the scripts you want to have support for?
please use [Wikipedia's Unicode block list](https://en.wikipedia.org/wiki/Unicode_block) page as a reference.
Thanks!

**Comment by AmirSimantov** [comment, marked as answer]
Closing. An issue #33 was added, instead.
